setup.py

```diff
@@ -1,13 +1,42 @@
-from setuptools import setup
-
-setup(
-    name="anekos",
-    author="NiumXp",
-    url="https://github.com/Nekos-life/Async-nekos.life-wrapper",
-    version="1.0.0",
-    packages=["anekos"],
-    install_requires=["aiohttp>=3.6.2"],
-    license="MIT",
-    description="An unofficial async wrapper for nekos.life API!",
-    python_requires=">=3.6"
-)
+from setuptools import setup
+
+requirements = []
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
+readme = ''
+with open('README.rst') as f:
+    readme = f.read()
+
+setup(
+    name='anekos',
+    author='NiumXP',
+    project_urls={
+        'Website': 'https://nekos.life',
+        'Issue tracker': 'https://github.com/Nekos-life/Async-nekos.life-wrapper',
+    },
+    version='1.0.6',
```

Comment:
Can you add a code to get version from `anekos.__init__.py`?

Something like it:
```python
import re

with open("anekos.__init__.py") as file:
    version = re.search(r"^__version__\s*=\s*[\"']([^\"']*)[\"']", code, re.MULTILINE).group(1)
```

Reply:
done